src/Fixtures/index.js

```diff
@@ -1410,33 +1705,112 @@ const firstWeekFitnessTraining = {
 const coed2MarkTraining = {
   id: 3,
   illustrationType: 'Martin',
-  title: 'Coed 2 Mark',
+  title: 'Build your defense',
   image: 'https://zupimages.net/up/20/19/su0s.jpg',
   description:
     "This training session aims at practicing your mark skills. Don't be broken to easily, generate dangerous passes",
-  drills: [warmupGameDrill, chevronDrill, markDrill, breakDrill, game3v3],
+  drills: [warmupGameDrill, dwarfDuelDrill, kyeDrill, breakDrill, normalGame],
 };
 
 const coed2LongTraining = {
   id: 4,
   illustrationType: 'Martin',
```

Comment:
Do we need this attribute on trainings? I think we could remove it

Or maybe call it `author`, it looks like a mistake